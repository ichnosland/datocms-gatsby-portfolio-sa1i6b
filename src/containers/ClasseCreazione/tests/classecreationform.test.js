import React from 'react';
import { shallow } from 'enzyme';

import ClasseCreazioneForm from '../ClasseCreationForm';

const mockProps = {
  data: {
    display: '',
    creaNuovaClasse: false,
    scuolaSelezionata: {
      pk: 2,
      nome: 'nome',
      classi: [],
    },
    nuova: false,
  },
  display: '',
  product: 'product',
};

describe('<ClasseCreazioneForm />', () => {
  it('should match snapshot', () => {
    const renderedComponent = shallow(
      <ClasseCreazioneForm {...mockProps} />
    );

    expect(renderedComponent).toMatchSnapshot();
  });
});
