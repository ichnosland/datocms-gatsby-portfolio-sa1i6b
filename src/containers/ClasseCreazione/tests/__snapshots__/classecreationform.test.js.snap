// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ClasseCreazioneForm /> should match snapshot 1`] = `
<Connect(Form(ClasseCreazioneForm))
  data={
    Object {
      "creaNuovaClasse": false,
      "display": "",
      "nuova": false,
      "scuolaSelezionata": Object {
        "classi": Array [],
        "nome": "nome",
        "pk": 2,
      },
    }
  }
  destroyOnUnmount={true}
  display=""
  enableReinitialize={false}
  forceUnregisterOnUnmount={false}
  form="creazioneClasse"
  getFormState={[Function]}
  keepDirtyOnReinitialize={false}
  persistentSubmitErrors={false}
  product="product"
  pure={true}
  shouldAsyncValidate={[Function]}
  shouldError={[Function]}
  shouldValidate={[Function]}
  shouldWarn={[Function]}
  touchOnBlur={true}
  touchOnChange={false}
  updateUnregisteredFields={false}
/>
`;
